common/app/mvt/MultiVariateTesting.scala

```diff
@@ -76,6 +76,18 @@ object ABJavascriptRenderingControl extends TestDefinition(
   def canRun(implicit request: RequestHeader): Boolean = participationGroup.contains("control")
 }
 
+object ABImageTestService extends TestDefinition(
```

Comment:
Maybe add a comment above this switch just to remind anyone removing this switch to also remove all the implicit params. I would just link to the PR. This is just in case anyone else is removing this switch just in case its not you.

Reply:
Removing the switch would mean the `ImageService` object doesn't compile anymore. So whoever who does it would be pointed toward the code that would need to be remove alongside the switch I think

Reply:
Fair enough!